include/revng/Support/Error.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Error.h"
+
+namespace revng {
+
+template<typename... Ts>
+inline llvm::Error createLLVMError(char const *Fmt, const Ts &...Vals) {
```

Comment:
I would call this `revng::createError` or `revng::createStringError`, it's not something we only use with LLVM APIs.

Reply:
The reason I put `LLVM` in there is so that it is easy to replace :wink:

That being said, do you want to keep it in `revng` namespace? We don't use it that much after all. I only added it to differentiate from the LLVM alternative.

Reply:
Ah, that's what you meant. The `llvm` was there more to reflect that it makes an `llvm::Error`, and not about where it's supposed to be used. That being said, in hindsight, I can clearly see your point!

Let's go with `revng::createError`.

Reply:
Done.